rfcs/text/0000-planning-for-the-dcp.md

```diff
@@ -0,0 +1,209 @@
+
+One of the **challenges** facing the DCP as it matures was identified in this [Update from HCA Science Governance](https://docs.google.com/presentation/d/13hZ7vmr371mN1YFjFyDN-6c0Ul95NwiJcVsjA3nJFVE/edit#slide=id.p3):
+
+> "A lack of reproducible planning process that prioritizes deliverables and ensures confidence that they will meet the needs of HCA"
+
+### DCP _data as a service_
+
+As a starting point, DCP should think *outside-the-box* and adopt one of the [principles of LeSS](https://less.works/less/principles/whole-product-focus.html) 
+>"Customers don’t buy a part of the product, but the whole product … there is no value in separate parts or half-working parts."
+
+Historically, ownership of the DCP software has been distributed across our canonical architectural diagram with its colorful *think-inside-the-boxes* (components). As a result, DCP planning and development have also been distributed to a high degree, resulting in questions or confusion about DCP priorities relative to component priorities with a subsequent lack of alignment for delivery of product increments, unless externally imposed by requests from HCA Science Governance. 
+
+For example, how does a developer know whether to prioritize research for an operational scoping task assigned by **Technical Architecture**, collaborate on a cross-component RFC surfaced at a DCP quarterly meeting, or develop a component-specific feature? 
+
+To best serve our users and address the concerns of HCA Science Governance, our plans must focus on the needs and priorities of DCP *data as a service* rather than the individual *Chartreuse* or *Polka Dot* components. This requires a DCP-wide project management process with clear priorities and product increments delivered on a regular cadence. 
+
+## Scientific "guardrails" 
+
+As key stackholders, the *Oversight Committee* MUST attend Sprint Reviews on a quarterly basis to offer feedback. 
+
+## Detailed Design or _Scrum meets ZenHub_
+
+### **!** Scrum
+
+A single agile framework (referenced as **!** **Scrum**) MUST be used by all component teams for planning and developing DCP *data as a service* in an iterative, incremental manner.
+
+Based on the broad adoption of Scrum variations across the DCP community, the expectation is that the reader is familiar with this framework and its artifacts. If not, please review the [definitive reference](https://scrumguides.org/docs/scrumguide/v2017/2017-Scrum-Guide-US.pdf) before proceeding.
+
+Exceptions to Scrum (colloquially known as [ScrumButs](https://www.scrum.org/resources/what-scrumbut) in that community) are described below. 
+
+As identified by charters, the DCP **Product Owners** (with guidance from **UXR**) are the **!** **Scrum** Product Owners in violation of Scrum’s *"the product owner is one person, not a committee"*.)
+
+There is also no formal DCP **Scrum Master**.
+
+### ZenHub
```

Comment:
I would describe it more as a project management application. It's not limited to task tracking since we also use it for prioritization, reporting, etc.  This would echo the line in the **Summary**:  

> This RFC adopts and formalizes existing best project management practices and applications